stubs/pywin32/win32/win32api.pyi

```diff
@@ -248,6 +248,9 @@ def Sleep(time, bAlterable: int = ..., /): ...
 def TerminateProcess(handle: int, exitCode: int, /) -> None: ...
 def ToAsciiEx(vk, scancode, keyboardstate, flags: int = ..., hlayout: Incomplete | None = ..., /): ...
 def Unicode() -> str: ...
+@overload
+def UpdateResource(handle: Never, type: Never, name: Never, data: str, language: Never, /) -> NoReturn: ...
```

Comment:
What's the purpose of this overload?

Reply:
Oops! This must've been leftover. I've re-reviewed my PR and done some small adjustments.